src/sdk/pynni/nni/nas/pytorch/callbacks.py

```diff
@@ -0,0 +1,69 @@
+import json
+import logging
+import os
+
+import torch
+
+_logger = logging.getLogger(__name__)
+
+
+class Callback:
```

Comment:
not a good name, better to make it clear that callback for what

Reply:
The design is from: https://github.com/keras-team/keras/blob/master/keras/callbacks/callbacks.py. Only a course version here. Will refactor later.